include/odp/api/spec/crypto.h

```diff
@@ -131,6 +131,12 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_AES_GCM,
 
+	/** AES in Galois/Counter MAC Mode
+	 *
+	 *  @note Must be paired with cipher ODP_CIPHER_ALG_NULL
+	 */
+	ODP_AUTH_ALG_AES_GMAC,
```

Comment:
This should specify that GMAC authentication is done for packet payload (instead of AAD). NIST/RFC documents use term AAD for the data. So, add here (or somewhere in the file) a reference to e.g. 
https://csrc.nist.gov/publications/detail/sp/800-38d/final, and document that AAD there means packet payload in ODP with GMAC.

For other algos AAD is the same as NIST/RFC specs.

Reply:
I think this needs to be still a bit more specific. For example:

"NIST and RFC specifications about GCM/GMAC refer to all data to be authenticated as AAD. In constrast to that, ODP API specifies the bulk of authenticated data to be located in packet payload for all authentication algorithms - including GMAC."

The first '@note' doxygen tag can be removed.

Also do we support still AAD in GMAC case? I think we should if there's no reason not to support. So, bulk of data in packet, additional data possible through AAD (like any other algo).

Reply:
There is no AAD for other MAC algorithms, so AAD is not supported for GMAC.

Reply:
@psavol updated description